doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -205,6 +205,8 @@ Bluetooth LE
   This was introduced in |NCS| 2.7.0 as experimental.
   The :ref:`bt_scanning_while_connecting` sample showcases how you can use this feature to reduce the time to establish connections to many devices.
 
+* Fixed an issue where Bluetooth applications built with the ``nordic-ble-rpc`` snippet (in the :ref:`ble_rpc` configuration) did not work on the nRF54H20 devices due to incorrect memory mapping.
```

Comment:
This entry needs to be removed from this file as its not part of the 2.9 release.